.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -28,21 +32,16 @@ const getPipeline = (filename: string, removeSteps = true) => {
 };
 
 (async () => {
+  const pipeline: string[] = [];
+
   try {
     const skippable = await areChangesSkippable(SKIPPABLE_PR_MATCHERS, REQUIRED_PATHS);
 
     if (skippable) {
-      console.log('All changes in PR are skippable. Skipping CI.');
-
-      // Since we skip everything, including post-build, we need to at least make sure the commit status gets set
-      execSync('BUILD_SUCCESSFUL=true .buildkite/scripts/lifecycle/commit_status_complete.sh', {
-        stdio: 'inherit',
-      });
-      process.exit(0);
+      console.log(emptyStep);
```

Comment:
Are we able to return without the log?  We're still going to get a build success comment, I don't think we need to spin up an agent to echo.

Reply:
I'm afraid, if we're not emitting anything, the wrapping command will fail with an empty input stream:
```
https://buildkite.com/elastic/kibana-pull-request/builds/222586#0190cace-0e1d-4c8a-8acb-3fc47a0b42f6
```
This is the command that runs any pipeline script:
```
chmod u+x .buildkite/scripts/pipelines/pull_request/pipeline.sh && .buildkite/scripts/pipelines/pull_request/pipeline.sh | buildkite-agent pipeline upload
```

We can maybe try a yaml that's `steps: []`

Reply:
Yes, this works, thanks for the suggestion!